Shoko.Server/API/v2/Modules/PlexWebhook.cs

```diff
@@ -179,10 +184,31 @@ private void Scrobble(PlexEvent data, SVR_JMMUser user)
                 break;
         }
 
+        var key = metadata.Key.Split("/").LastOrDefault();  // '/library/metadata/{key}'
+
+        var plexEpisode = (SVR_Episode)GetPlexEpisodeData(key, user);
+        var episode = plexEpisode?.AnimeEpisode;
```

Comment:
Can we add this with a guard against people not having a valid plex connection?

The original design was that the webhook did not require user plex login, so it was entirely working off it's own data.

Happy to add this in, but I think it should be at least guarded to allow for users to not have their own plex config.

Reply:
Can you clarify the scenario you're describing? The user has plex webhooks enabled, but hasn't linked their plex account to Shoko?

If that's the case, I suppose I could fall back original behavior of only matching episodes.